src/js/controller/controls-loader.js

```diff
@@ -1,19 +1,31 @@
 import { chunkLoadWarningHandler } from '../api/core-loader';
+import { OS } from '../environment/environment';
 
 let controlsPromise = null;
 
 export const ControlsLoader = {};
 
 export function loadControls() {
     if (!controlsPromise) {
-        controlsPromise = require.ensure(['view/controls/controls'], function (require) {
-            const ControlsModule = require('view/controls/controls').default;
-            ControlsLoader.controls = ControlsModule;
-            return ControlsModule;
-        }, function() {
-            controlsPromise = null;
-            chunkLoadWarningHandler(301130)();
-        }, 'jwplayer.controls');
+        if (OS.tizen) {
+            controlsPromise = require.ensure(['view/controls/tizen/tizen-controls'], function (require) {
+                const ControlsModule = require('view/controls/tizen/tizen-controls').default;
+                ControlsLoader.controls = ControlsModule;
+                return ControlsModule;
+            }, function() {
+                controlsPromise = null;
+                chunkLoadWarningHandler(301130)();
```

Comment:
There should be a different error code per file (so we know it's jwplayer.controls.tizen.js that failed to load).